app/lib/actions/katello/content_view_puppet_module/destroy.rb

```diff
@@ -11,7 +11,7 @@ def plan(repository)
           repository.puppet_modules.each do |puppet_module|
             # first, process content view puppet modules that have been specified by version
             library_repos = ::Katello::Repository.in_environment(repository.organization.library).
-                                                  where(:pulp_id => puppet_module.repoids)
+              where(:pulp_id => puppet_module.repositories.map(&:pulp_id))
```

Comment:
You mean re-indent it? I was following the style shown in lines 25-27, 30-31, etc. of this file.